src/Common/MorganStanley.ComposeUI.Common.BasicModels/MorganStanley.ComposeUI.Common.BasicModels.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<!--
```

Comment:
This package seems unnecessary, should be merged into a `MorganStanley.ComposeUI.Core` package with all the abstractions and types commonly used by all plugins.

Reply:
I'd like Core to be a folder containing re-usable and UI framework independent projects, so let us come up with a different name for the project containing ViewModel basic functionality.

Reply:
For now I left it as it is, but placed it within Core folder

Reply:
Open for renaming suggestions as part of another Jira.